### DISCUSSION TITLE
Can't Find Error Message in Codebase

### DISCUSSION THREAD

**Discussion by stefkuk** [triplea-game/triplea#12424] [Q&A] [answered]
When you open a tsvg file on MacOS, an error pops up:
<img width="656" alt="Screenshot 2024-03-09 at 7 47 17 PM" src="https://github.com/triplea-game/triplea/assets/45446864/00745079-6bc7-4f3e-9b74-90ac87f95992">

I've searched the codebase but can't find this specific text anywhere in the codebase. Can anyone help me figure out what code generates this error message? Even doing a search for "Unsupported feature" (the title of the text box) doesn't show anything.

**Comment by DanVanAtta** [comment, marked as answer]
To find the error message - switch to git tag '2.5.22208'
